browser-extension/common/src/custom-elements/test-rule-widget/explicit-test-rule-widget/index.ts

```diff
@@ -63,8 +67,8 @@ class RQExplicitTestRuleWidget extends RQTestRuleWidget {
   }
 
   showRuleAppliedStatus(appliedStatus: boolean) {
-    const ruleStatusContainer = this.shadowRoot.getElementById("rule-status");
-    const minimizedStatusBtn = this.shadowRoot.getElementById("minimized-status-btn");
+    const ruleStatusContainer = this.shadowRoot.getElementById("rq-rule-status");
+    const minimizedStatusBtn = this.shadowRoot.getElementById("rq-minimized-status-btn");
```

Comment:
```suggestion
    const minimizedStatusBtn = this.shadowRoot.getElementById("rq-test-rule-minimized-btn");
```
WDYT? `rq-minimized-status-btn` seems generic